core/src/br/cefetmg/games/screens/BaseScreen.java

```diff
@@ -201,6 +201,11 @@ public final void dispose() {
         }
         this.cleanUp();
     }
+
+    public BaseScreen getPrevious() {
```

Comment:
Isto não é necessário. Vi que você colocou isto porque achou que era necessário na hora de trocar a tela, mas a tela `previous` é simplesmente a tela atual (`this`), ou seja: `super.game.setScreen(new MenuScreen(...), this)`.